apps/meteor/tests/end-to-end/api/chat.ts

```diff
@@ -1460,6 +1499,42 @@ describe('[Chat]', () => {
 					expect(res.body.message).to.have.property('attachments').that.is.an('array').that.has.lengthOf(0);
 				});
 		});
+
+		describe('Bad words filter', () => {
+			before(async () => {
+				await updateSetting('Message_AllowBadWordsFilter', true);
```

Comment:
I think we should be doing the before and the after settings update inside of a promise all, check the comment on the other test